content/zh-cn/contribute/_index.md

```diff
@@ -6,101 +6,116 @@ menu:
-3) 帮助翻译词汇表
+1) [在现有的议题上工作](#在现有的议题上工作)
+2) [提出新术语](#提出新术语)
+3) [更新现有术语](#更新现有术语)
+4) [云原生词汇表翻译帮助](#云原生词汇表翻译帮助)
 
 ## 加入 Glossary 社区! 
-如果您想定期投稿, 请考虑加入我们的每月词汇表工作组会议. 您可以在 [CNCF calendar](https://www.cncf.io/calendar/) 找到会议详情. 您还可以在 CNCF Slack 上的 #glossary 频道中与维护者和其他贡献者联系 — 期待美好相遇! 
+如果你想定期做出贡献，可以考虑加入我们每月的词汇工作组会议。你可以在 [CNCF 日历](https://www.cncf.io/calendar/) 中找到会议细节。你也可以在 CNCF Slack 的 [#glossary](https://cloud-native.slack.com/archives/C02TX20MQBB) 频道中与维护者和其他贡献者联系，我们很乐意见到你! 
 
-## 提出新术语
-您可以提出一个新术语供其他人使用或自己创建一个新定义. 无论哪种方式, 您都将从创建问题开始(请注意, 您需要一个 GitHub 帐户来执行此操作).
+## 在现有的议题上工作
+进入云原生词汇表 GitHub 仓库的 [议题](https://github.com/cncf/glossary/issues)。在那里你会看到一个所有议题的列表。你可以通过标签来过滤（例如，English language, help needed, good first issue）。请注意，你需要一个 GitHub 账户来做这些事情。
+
+! [Issue 和 labels](/images/how-to/issue-and-labels.png)
+
+确保你感兴趣的术语还没有分配给某人。这里你可以看到，前三个术语是可用的，而下一个术语已经被分配了：
+
+![分配一个术语](/images/how-to/howto-04.png)
 
-以下是针对尚未熟悉 GitHub 的人的分步指南. **如果你是 GitHub Pro 账户**, 请快速查看以确保您使用我们的问题模板、适当的命名约定、在 Slack 上声明 PR（否则我们可能会错过它）以及在哪里可以找到文件模板. 在开始之前, 请务必阅读 [Style Guide](/zh-cn/style-guide/).
+一旦你找到了一个你想做的术语，就在议题中说出来。点击它并添加一个评论。
 
-### 创建 issue
-前往 [Glossary GitHub repo](https://github.com/cncf/glossary/issues) 并点击 "new issue".
+![提出一个议题](/images/how-to/claiming-an-issue.png)
+
+此外，请加入 CNCF Slack 的 [#glossary](https://cloud-native.slack.com/archives/C02TX20MQBB) 频道，让维护者知道你为一个新术语提出了一个议题（最好标记 _@Catherine Paganini_, _@jmo_, _@Seokho Son_, _@Jihoon Seo_, 和/或 _@iamnoah_ 以确保他们不会错过它）。请注意，你一次只能认领一个术语。如果你想做多个术语，请在申请下一个术语之前完成前一个术语。
+
+一旦他们把它分配给你，你就可以开始工作了。接下来的步骤，请参考 [提交一个新术语（创建一个PR）](#submitting-a-new-term-creating-a-pr)部分。
+
+## 提出新术语
+你可以提出一个新的术语让别人去研究，或者自己创造一个新的定义。无论哪种方式，你都要从创建一个议题开始。请注意，术语必须符合 [CNCF的云原生定义]（https://github.com/cncf/toc/blob/main/DEFINITION.md）。唯一的例外是理解云原生概念所需的基础性术语。
 
-![issues](/images/how-to/howto-01.png)
+以下是为那些还不熟悉 GitHub 的人提供的分步指南。**如果你是 GitHub 的专业人员**，请**特别**看一下，确保你使用我们的议题模板，适当的命名惯例，在 Slack 上声明一个 PR（否则我们可能会错过它），以及在哪里找到文件模板。在开始之前，请确保阅读 [风格指南](/style-guide/)--谢谢! 
 
-您将看到各种模板. 要使用中文提出新术语, 请选择“Chinese Language Glossary Request”.
+### 创建议题
+进入云原生词汇表 GitHub 仓库的 [议题](https://github.com/cncf/glossary/issues)，点击 "new issue"。
 
-![templates](/images/how-to/howto-02.png)
+![议题](/images/how-to/howto-01.png)
 
-添加您建议的单词并点击“submit new issue”. 如果您只是提出一个新术语, 那么您就完成了! 要处理它, 请按照以下步骤操作.
+你会看到各种各样的模板。要提出一个中文的新术语，请选择 "Chinese Language Glossary Request"。
 
-![new issue](/images/how-to/howto-03.png)
+![模板](/images/how-to/english-issue-template.jpg)
 
-请加入 CNCF Slack 上的 #glossary 频道, 让@Catherine Paganini、@jmo 和@Seokho Son 知道您已经提交了一个问题并希望解决它. 他们会将问题分配给您, 向其他所有人发出该术语已被占用的信号.
+添加你所建议的词，回答下面的两个议题，勾选复选框，然后点击 "submit new issue"。如果你只是提出一个新词，你就完成了! 要进行工作，请按照接下来的步骤进行。
 
-在这里, 您可以看到前三个术语可用, 而下一个术语已分配给某人.
+### 分流你的议题
+接下来，维护者将对该议题进行分流。这意味着他们将评估该术语是否应成为术语表的一部分（注意，不是每个术语都会被接受。术语应该是既定的、广泛使用的云计算原生术语）。
 
-![assigning a term](/images/how-to/howto-04.png)
+请让维护者知道你已经在 Slack 上提交了一个术语，否则他们可能会错过它。最好是给_@Catherine Paganini_、_@jmo_、_@Seokho Son_、_@Jihoon Seo_和/或_@iamnoah_打上标签。如果该术语被批准，并且你想从事该工作，他们会将其分配给你。
```

Comment:
Preview: https://deploy-preview-881--cncfglossary.netlify.app/zh-cn/contribute/#分流你的议题

![image](https://user-images.githubusercontent.com/46767780/167806981-f724cfa3-a7da-423d-a154-3dd84abb53db.png)

The Markdown italic is not working here, since
1) Usually no spaces are used in Chinese sentences, and
2) the Markdown renderer does not handle the Chinese comma (`、`) and the Chinese period (`。`) as a sort of punctuation marks, but just as normal characters or alphabets.

[We may try asterisk (`*`) instead of underscore (`_`) to implement italic](https://www.markdownguide.org/basic-syntax/#italic),
but my test on https://markdownlivepreview.com was not successful with asterisk either.

So, how about we use HTML tags to implement italic here?

Plus, is it good or not to put a space between:
- `最好是给` and `@Catherine Paganini`
- `@Jihoon Seo` and `和/或`
- `和/或` and `@iamnoah`
- `@iamnoah` and `打上标签`

?
This is up to you, Chinese L10n team members. 😊

```suggestion
请让维护者知道你已经在 Slack 上提交了一个术语，否则他们可能会错过它。最好是给<em>@Catherine Paganini</em>、<em>@jmo</em>、<em>@Seokho Son</em>、<em>@Jihoon Seo</em>和/或<em>@iamnoah</em>打上标签。如果该术语被批准，并且你想从事该工作，他们会将其分配给你。
```

---

Edit: this is another available option, and this option might be the better one!

```suggestion
请让维护者知道你已经在 Slack 上提交了一个术语，否则他们可能会错过它。最好是给 _@Catherine Paganini_, _@jmo_, _@Seokho Son_, _@Jihoon Seo_, 和/或 _@iamnoah_ 打上标签。如果该术语被批准，并且你想从事该工作，他们会将其分配给你。
```